rules/rules.css

```diff
@@ -0,0 +1,61 @@
+@import url(../landingpage/landingpage.css);
+
+
+.rules {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 100vh;
+    background-color: var(--primary-blue);
+
+}
+
+.rules-box {
+    padding: 4rem;
+    text-align: center;
+    border-radius: 2rem;
+
+}
+
+.rules-box h2 {
+    color: var(--secondry-orange);
+}
+
+.rule-points {
+    display: flex;
+    align-items: center;
+    gap: 1rem;
+    padding: 1rem;
+    background-color: var(--third-white);    
+    border-radius: 2rem;
+    margin: 1rem;
+
+
```

Comment:
If you have time, remove these additional lines. These are just minute details, nothing to worry about.

Reply:
done :)